### ISSUE TITLE
Armor loose loot doesnt have removeable plates installed

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#618]
## **General information**:

**Project issue is for:** </br>
Server

**Expected result**:</br>
Armor loose loot has removeable armor plates installed

**Actual result**:</br>
Armor loose loot has soft inserts but not removeable armor plates installed

Maps reproduceable on: Ground Zero

## **How to reproduce**:
1. Spawn in a map
2. Find armor as loose loot
3. See that armor doesn't have removeable inserts installed

---
### Attachments
- [server-2024-01-23.log](https://dev.sp-tarkov.com/attachments/76dc1654-91f1-49c8-a3c2-39a874e987de) (468 KiB)
- [LogOutput.log](https://dev.sp-tarkov.com/attachments/8e06ef69-8887-4979-8030-c49c111ea732) (61 KiB)
- [2024.01.24_0-21-45_0.14.0.0.28375 traces.log](https://dev.sp-tarkov.com/attachments/6e709a6c-86db-41e2-9c1e-7f655026db6a) (115 KiB)
- [Screenshot 2024-01-23 195516.png](https://dev.sp-tarkov.com/attachments/47e0997e-5097-4d8d-a6cd-86461ef70694) (787 KiB)

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/337) by _**r2go**_

**Comment by refringe** [comment]
ItemHelper.ts -> addChildSlotItems()


```
            // Roll chance for non-required slot mods
            if (modSpawnChanceDict && !slot._required)
            {
                // only roll chance to not include mod if dict exists and has value for this mod type (e.g. front_plate)
                const modSpawnChance = modSpawnChanceDict[slot._name.toLowerCase()];
                if (modSpawnChance)
                {
                    if (!this.randomUtil.getChance100(modSpawnChanceDict[slot._name]))
                    {
                        continue;
                    }
                }
            }
```

```
back_plate:
50
front_plate:
50
mod_equipment_000:
5
mod_equipment_001:
5
mod_equipment_002:
5
mod_mount:
5
mod_nvg:
5
```
maybe armor is just super unlucky in getting actual plates in there.

This probably isnt a bug looking at the code, people are just getting unlucky. The distribution for getting armor with no plates in Live seems extremely low as compared to this 50/50.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/337#issuecomment-21941) by _**r2go**_

**Comment by refringe** [comment]
This was bugged, but should be fixed with [3ec63ef31d](https://dev.sp-tarkov.com/SPT-AKI/Server/commit/3ec63ef31dd0df43fac2ce9cfc5cf18a71f9a2c0).

At that point in the code, the slots are named `Back_plate` and `Front_plate`. Because of this, they weren't properly accessing the modSpawnChanceDict.


(I also wish I'd pulled before I did the troubleshooting myself.)

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/337#issuecomment-21946) by _**Spoony**_

**Comment by refringe** [comment]
I've adjusted the loose loot plate insertion chance to be 75%, should help

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/337#issuecomment-22062) by _**chomp**_
